.github/workflows/test-batch-processing.yml

```diff
@@ -0,0 +1,54 @@
+name: Test batch_processing.sh
+
+on:
+  push:
+    branches:
+      - master
+      - release
+  pull_request:
+    branches:
+      - '*'
+  schedule:
+    # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onschedule
+    # > Scheduled workflows run on the latest commit on the default or base branch
+    # i.e. this can only run on master
+    - cron:  '0 11 * * *'
+
+env:
+  # Turns on color output for pytest. See: https://github.com/pytest-dev/pytest/issues/7443#issuecomment-656642591
+  PY_COLORS: "1"
+  # Used in 'pytest.mark.skipif()' for test, which prevents the test from being run during the usual test suite
+  BATCH_PROCESSING_CI_JOB: "1"
+
+jobs:
+  batch_processing:
+    name: Test batch_processing.sh
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04 ]  # TODO: Change to [ ubuntu-18.04, macos-10.15 ]
+                              # macOS currently fails due to https://github.com/neuropoly/spinalcordtoolbox/issues/3194
```

Comment:
I've removed `macos-10.15` in b978a6e, because I figure it will only be useful when the discrepancies are fixed.